tests/dag_processing/test_processor.py

```diff
@@ -635,6 +636,30 @@ def test_import_error_tracebacks(self, tmpdir):
             )
             session.rollback()
 
+    def test_process_file_should_remove_unknown_serialized_dags(self):
+        dag_file = os.path.join(
+            os.path.dirname(os.path.realpath(__file__)), '../dags/test_only_dummy_tasks.py'
+        )
```

Comment:
```suggestion
        dag_file = os.path.join(TEST_DAGS_FOLDER, 'test_only_dummy_tasks.py')
```
Then import `TEST_DAGS_FOLDER` from `tests.models`

Reply:
nice, thanks